src/index.html

```diff
@@ -2,13 +2,88 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <div class="picture">
+        <a href="#" >
+          <img src="./images/logo.png" alt="Logo">
+        </a>
+      </div>
+      <nav>
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#Apple"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#Samsung"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#Smartphones"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#Laptops & Computers"
+              class="nav_link"
+              data-qa="hover"
+             >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#Gadgets"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#Tablets"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#Photo"
+              class="nav_link"
+            >
```

Comment:
this tag can be written in one line provided that there are less than 3 attributes in the tag (it's up to you)